tensorflow/lite/kernels/non_max_suppression_test.cc

```diff
@@ -26,20 +27,106 @@ namespace {
 
 using ::testing::ElementsAreArray;
 
+using FloatData = std::initializer_list<float>;
+
+static constexpr FloatData k_input_boxes_data{
+    1, 1,     0, 0,     // Box 0
+    0, 0.1,   1, 1.1,   // Box 1
+    0, .9f,   1, -0.1,  // Box 2
+    0, 10,    1, 11,    // Box 3
+    1, 10.1f, 0, 11.1,  // Box 4
+    1, 101,   0, 100    // Box 5
+};
+
+static constexpr FloatData k_input_scores_data{0.9f,  0.75f, 0.6f,
+                                               0.95f, 0.5f,  0.3f};
+
+static constexpr FloatData k_ik_input_iou_threshold_data{0.5f};
+
+static constexpr FloatData k_input_score_threshold_data{0.0f};
+
+static constexpr FloatData k_input_sigma_float_data{0.5f};
+
+using Int8Data = std::initializer_list<int8_t>;
+static constexpr Int8Data k_input_sigma_int8_data{0};
+
+using Int16Data = std::initializer_list<int16_t>;
+static constexpr Int16Data k_input_sigma_int16_data{16092};
+
+static const auto k_min_max_input =
+    std::minmax_element(k_input_boxes_data.begin(), k_input_boxes_data.end());
+
 class BaseNMSOp : public SingleOpModel {
  public:
```

Comment:
```suggestion
 public:
   virtual ~BaseNMSOp() {}
 
```

I think this should fix the current issue